pkg/rc/replication_controller.go

```diff
@@ -892,10 +895,16 @@ func (rc *replicationController) watchHealth() bool {
 
 	for {
 		select {
-		case err := <-errCh:
-			rc.logger.WithError(err).Errorln("Node transfer health checker sent error")
+		case err, ok := <-errCh:
+			// TODO rewrite once we understand why nil errors were sent to this chan
```

Comment:
I know this looks weird and I'm not actually using the `ok` from  `err, ok`. But I've left it this way _just in case_ the nil error did not actually come from a closed channel. Once we learn why the error was nil I'll rewrite this.